python/tvm/relay/build_module.py

```diff
@@ -607,16 +607,16 @@ class AotExecutor(_interpreter.Executor):
     device : :py:class:`Device`
         The runtime device to run the code on.
 
-    target : :py:class:`Target`
-        The target option to build the function.
+    raw_targets : Array[tvm.target.Target]
+        The available targets.
     """
 
-    def __init__(self, mod, device, target):
+    def __init__(self, mod, device, raw_targets):
         assert mod is not None
         self.mod = mod
         self.device = device
-        self.target = target
-        assert target.attrs.get("executor", "graph") == "aot"
+        self.raw_targets = raw_targets
+        assert raw_targets[0].attrs.get("executor", "graph") == "aot"
```

Comment:
shall we add a corresponding assert for graph?

what's the expected form of this array of Targets (seems the host should be the first one)? how should we know what to supply here (and for GraphExecutor)?

Reply:
See following comment.